### ISSUE TITLE
po2json handles braces wrong.

### ISSUE THREAD

**Issue by FinnStutzenstein** [OpenSlides/OpenSlides#4185]
In the german po file:
```
msgid "Your login for {event_name}"
msgstr "Zugangsdaten für {event_name}"
```
The resulting JSON:
```
"Your login for {event_name}":"Zugangsdaten für \\{event_name\\}"
```
Maybe related to https://github.com/mikeedwards/po2json/pull/78.
Breaks stuff here: https://github.com/OpenSlides/OpenSlides/pull/4177
Visible in Openslides: settings->participants->email->help texts for message body and subject.

**Comment by MaximilianKrambach** [comment]
Also the string %user% does not come out again as it should (I can't remember exactly how; I wrote it into a different issue yesterday, but cannot find it)

**Comment by MaximilianKrambach** [comment]
`The user %user% has no email,...` is the (only?) string affected.

**Comment by tsiegleauq** [comment]
Has been temporarily fixed by @MaximilianKrambach in #4196.
We should keep this issue open to track the development regarding this on po2json.

**Comment by jsangmeister** [comment]
po2json is no longer used
